app/src/test/java/com/eveningoutpost/dexdrip/G5Model/BatteryInfoRXTests.java

```diff
@@ -0,0 +1,108 @@
+package com.eveningoutpost.dexdrip.G5Model;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.nio.ByteBuffer;
+import java.nio.ByteOrder;
+
+/**
+ * Note that if there was documentation on range boundaries
+ * for the values in voltageA/B/resistance, temprature and
+ * so on, we could have a much MUCH more stable environment
+ * in a short time frame.
+ */
+public class BatteryInfoRXTests {
```

Comment:
A unit test for a class is typically named with the class name and the suffix "Test"; in this case: `BatteryInfoRxMessageTest`.